Source/AppScaffolding/LibationScaffolding.cs

```diff
@@ -296,21 +298,20 @@ private static (Octokit.Release, Octokit.ReleaseAsset) getLatestRelease(TimeSpan
 		}
 		private static async System.Threading.Tasks.Task<(Octokit.Release, Octokit.ReleaseAsset)> getLatestRelease()
 		{
-			var ownerAccount = "rmcrackan";
-			var repoName = "Libation";
+			const string ownerAccount = "rmcrackan";
+			const string repoName = "Libation";
 
 			var gitHubClient = new Octokit.GitHubClient(new Octokit.ProductHeaderValue(repoName));
 
 			//Download the release index
 			var bts = await gitHubClient.Repository.Content.GetRawContent(ownerAccount, repoName, ".releaseindex.json");
 			var releaseIndex = JObject.Parse(System.Text.Encoding.ASCII.GetString(bts));
 			var regexPattern = releaseIndex.Value<string>(ReleaseIdentifier.ToString());
+			var regex = new System.Text.RegularExpressions.Regex(regexPattern, System.Text.RegularExpressions.RegexOptions.IgnoreCase);
 
-			// https://octokitnet.readthedocs.io/en/latest/releases/
-			var releases = await gitHubClient.Repository.Release.GetAll(ownerAccount, repoName);
+			//https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release
+			var latestRelease = await gitHubClient.Repository.Release.GetLatest(ownerAccount, repoName);
```

Comment:
Unless you make a new release with a lower version number than a previous release, this is equivalent (with >200 fewer API calls).

Reply:
Woah! That's a large difference -- noice